Assets/Scripts/KuriTextManager.cs

```diff
@@ -52,6 +57,29 @@ public enum PRIORITY {
             audioSource.volume = 0.05f;
         }
 
+        public void ToggleGlow()
+        { //toggles glow on and off by changing material, helper method for chest lights
```

Comment:
short form brackets eg

```C#
public void ToggleGlow(){
...
}
```
as opposed to long form (note long form is traditionally used for C# but I don't use it for this project
```C#
public void ToggleGlow()
{
...
}
```